### ISSUE TITLE
[Slim] Training accuracy is increasing very slowly

### ISSUE THREAD

**Issue by redserpent7** [tensorflow/tensorflow#11373]
Hi,

I feel something odd is happening with my Inception_Resnet_v2 training. I am what I'm seeing is correct but it seems the training will take forever at the current rate. 

I am training slim against the full Imagenet dataset approx 14M images. The training is running on 8GPUs and the batch size is the default 32. If my calculations are correct this means that each epoch should consist of approximately (54,000 steps):

14,000,000 / (8 * 32) 

The training has been running for 800,000 steps so far (approx 15 epochs) yet the accuracy just reached 22%.

I have been evaluating the model after each 100,000 steps and at the start, the accuracy used to increase by 5% after each 100,000 step up until 500,000 steps, yet now it seems to be rapidly slowing down as now I am getting in approximate 0.8% increase every 100,000 steps after reaching the 500,000 steps mark.

I am not sure if this is expected as I do not have any data to compare it with but at this rate it can take several months, maybe even years to reach a viable accuracy rate. I am trying to reach somewhere above 70% at least.

Now is this expected? Are there any ways to speed up training beside adding GPUs? And if the latter is my only option, is it possible to continue training on the same checkpoints once the number of GPUs has increased?

**Comment by michaelisard** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
